src/routes/documents/[id]-[slug]/sidebar/Notes.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import type { Document, Note } from "$lib/api/types";
+
+  import { _ } from "svelte-i18n";
+  import { Note16, Note24 } from "svelte-octicons";
+
+  import Empty from "@/lib/components/common/Empty.svelte";
+  import SidebarGroup from "@/lib/components/sidebar/SidebarGroup.svelte";
+  import SidebarItem from "@/lib/components/sidebar/SidebarItem.svelte";
+
+  import { canonicalUrl } from "$lib/api/documents";
+  import { noteUrl } from "$lib/api/notes";
+
+  export let document: Document;
+
+  $: notes = document.notes;
+  $: annotate = new URL("annotate/", canonicalUrl(document)).href;
+</script>
+
+<SidebarGroup>
+  <SidebarItem slot="title">
+    <Note16 />
+    {$_("sidebar.toc.notes")}
+  </SidebarItem>
+
+  <ol class="notes">
+    {#each notes as note}
+      <li>
+        <SidebarItem href={noteUrl(document, note).href} small>
+          {note.title}
```

Comment:
Long note titles need some overflow styling or else they break out of the container:

```suggestion
          <span class="note_title">{note.title}</span>
```

```css
.note_title {
  overflow: hidden;
  text-overflow: ellipsis;
}
```